packages/api-client/src/controllers/project/project.ts

```diff
@@ -64,26 +75,37 @@ export default class ProjectController {
 
   static async unlinkFork(
     request: UnlinkProjectRequest,
-    headers: any
-  ): Promise<any> {
-    return this.apiClient.put(
-      `/api/project/${request.projectId}/unlink-fork`,
-      request,
+    headers: Record<string, string>
+  ): Promise<UnlinkProjectResponse> {
+    return this.apiClient.delete(
+      `/api/project/${request.projectId}/fork`,
       headers
     )
   }
 
-  static async getForks(request: GetForkRequest, headers: any): Promise<any> {
-    return this.apiClient.get(
-      `/api/project/${request.projectId}/forks`,
-      headers
-    )
+  static async getForks(
+    request: GetForkRequest,
+    headers: Record<string, string>
+  ): Promise<[GetForkResponse]> {
+    let url = `/api/project/${request.projectId}/forks`
+    request.page && (url += `page=${request.page}&`)
+    request.limit && (url += `limit=${request.limit}&`)
+    request.sort && (url += `sort=${request.sort}&`)
+    request.order && (url += `order=${request.order}&`)
+    request.search && (url += `search=${request.search}&`)
```

Comment:
![image](https://github.com/user-attachments/assets/907f4d40-f85f-427f-9ab0-02c38d14b334)

We have just these few fields for get forks